packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -971,6 +971,12 @@ class FlutterPlugin implements Plugin<Project> {
         } // end def addFlutterDeps
 
         if (isFlutterAppProject()) {
+            project.tasks.create(name: "getBuildOptions") {
```

Comment:
A bit more than a nitpick on the name. These are not buildOptions they are buildVariants. Why not create a gradle task that uses the existing verbiage? 

Similarly since this does not return anything the task should be printVariants or listVariants.